nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -3,9 +3,9 @@
 with lib;
 
 let
-  cfg = config.services.bitwarden_rs;
-  user = config.users.users.bitwarden_rs.name;
-  group = config.users.groups.bitwarden_rs.name;
+  cfg = config.services.vaultwarden;
+  user = config.users.users.vaultwarden.name;
+  group = config.users.groups.vaultwarden.name;
```

Comment:
Systemd should sort out the permissions on `/var/lib/bitwarden_rs` automatically, so this shouldn't be a problem unless people go out of their way to modify the users in their own config, which I really hope no one does.

Reply:
Gosh you're right! I was mistaken.

So what is the incompatibility this PR introduces?

Reply:
People manually modifying the users/units in their configuration will have things explode because the unit is no longer there. In most cases it will explode in a pretty obvious way, so this is probably fine?

Reply:
Also anything that depends on the paths in the final derivations, like custom units and such.